test/unit/gpu/CMakeLists.txt

```diff
@@ -82,4 +82,9 @@ if (NOT RAJA_ENABLE_CLANG_CUDA)
     NAME scan
     SOURCES test-scan.cxx
     DEPENDS_ON gtest gtest_main ${CMAKE_THREAD_LIBS_INIT})
+
+#   raja_add_test(
```

Comment:
Unfortunately nvcc won't compile extended device-side lambdas if they're in a private/protected member function, which is how the test-fixture functions are implemented.